src/xo-app/new-vm/index.js

```diff
@@ -103,7 +103,7 @@ const LineItem = ({ children }) => (
 
 const Item = ({ label, children, className }) => (
   <span className={styles.item}>
-    {label && <span>{_(label)}&nbsp;</span>}
+    {label && <span>{label}&nbsp;</span>}
```

Comment:
My bad, I thought it was a common component :/

So I don't care about it either way, you can leave it like that.